src/utils/defaultNumberFormatter.ts

```diff
@@ -0,0 +1,15 @@
+import {format} from 'd3-format'
+
+const MAX_DECIMALS = 4
```

Comment:
Should this be configurable?

Reply:
Hmm maybe eventually as like a "precision to use when formatting numbers with the default/fallback value formatter" option. Seems a bit specific so I was hesitant to expose it as part of the public API.

I think right now the way for someone to specify how much precision they want would be to specify a value formatter for the column they care about.